packages/amplify-cli/src/extensions/amplify-helpers/path-manager.js

```diff
@@ -130,7 +130,14 @@ function getProviderInfoFilePath(projectPath) {
 
 function getBackendConfigFilePath(projectPath) {
   return path.normalize(path.join(
-    getDotConfigDirPath(projectPath),
+    getBackendDirPath(projectPath),
+    amplifyCLIConstants.BackendConfigFileName,
+  ));
+}
+
+function getCurrentBackendConfigFilePath(projectPath) {
+  return path.normalize(path.join(
+    getCurrentCloudBackendDirPath(projectPath),
     amplifyCLIConstants.BackendConfigFileName,
```

Comment:
should it be starting with capital `B`. Anything that starts with capital letter gives me an impression that is an class that can be initialized. If its a constant, it better to have the name in capital letters

Reply:
This is a higher level change. Right now we have this constants file with the fillowing info.

```
module.exports = {
  AmplifyCLIDirName: 'amplify',
  DotAmplifyDirName: '.amplify',
  DotConfigamplifyCLISubDirName: '.config',
  BackendamplifyCLISubDirName: 'backend',
  CurrentCloudBackendamplifyCLISubDirName: '#current-cloud-backend',
  PluginConfigFileName: 'plugin-config.json',
  ProjectConfigFileName: 'project-config.json',
  amplifyMetaFileName: 'amplify-meta.json',
  BrandName: 'AWS Amplify',
  CliName: 'amplify',
  DateTimeFormatString: 'YYYY-MM-DD-HH-mm-ss',
  DateTimeFormatStringCompact: 'YYYYMMDDHHmmss',
  DefaultAWSAccessKeyId: '<YOUR_ACCESS_KEY_ID>',
  DefaultAWSSecretAccessKey: '<YOUR_SECRET_ACCESS_KEY>',
  DefaultAWSRegion: 'us-east-1',
  LocalEnvFileName: 'local-env-info.json',
  ProviderInfoFileName: 'team-provider-info.json',
  BackendConfigFileName: 'backend-config.json',

};
```

We could decide as a team as to if we want to change all of these to caps since these are basically constants.